computer_vision/hog.py

```diff
@@ -0,0 +1,178 @@
+# Módulos
+
+import doctest
+import math
+import random
+from typing import List  # noqa: F401
+from typing import overload
+
+import cv2
+import numpy as np
+import numpy.typing as npt
+
+"""
+Histogram of oriented gradients
+https://en.wikipedia.org/wiki/Histogram_of_oriented_gradients
+"""
+
+# Establecemos una semilla
+random.seed(1234)
+
+
+class HistogramOrientedGradients:
+    def __init__(self, imagen: npt.NDArray[np.float32]):
+
+        """
+        imagen : numpy array of float values
+        """
+        self.imagen = imagen
+
+    @overload
+    def __setitem__(self, SupportsIndex, int) -> None:
+        ...
+
+    @overload
+    def __add__(self, List[float]) -> List[float]: \
```

Comment:
You have just indicated the type of the parameter. You need to also specify the name.
`def func(x: int)`